solr/solr-ref-guide/modules/query-guide/pages/analytics-mapping-functions.adoc

```diff
@@ -18,6 +18,8 @@
 
 Mapping functions map values for each Solr Document or Reduction.
 
+WARNING: The analytics component is deprecated. We recommend looking into similar functionality found in xref:json-facet-api.adoc[JSON Facet API].
```

Comment:
Maybe we should link to JIRA and say create a ticket if there's functionality you need that isn't covered by JSON facets?

Reply:
Yea, or ask them to write to users list, where we can qualify that there is in fact a need? Jira is invite-only so not a good experience for users to end up there and be able to do nothing 😕. 

Feel free to suggest some text and I’ll incorporate it.